extern/region-4.14/Makefile.in

```diff
@@ -1,7 +1,7 @@
-# Makefile.in generated by automake 1.16.1 from Makefile.am.
```

Comment:
@wmclaugh - do we care that we are now using a significantly older `automake` than was used previously (I think I've asked this before when @olaurino updated some of these dependencies)?

Reply:
@DougBurke - thanks for catching this. I've updated the PR to utilize 1.16.1 for consistency.